src/convertToObject.js

```diff
@@ -1,18 +1,16 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const temper = sourceString.split('\n').slice(1);
+  const result = {};
+
+  for (const ch of temper) {
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/97547824/06c75c4a-bfec-4880-ab19-3f428a0b71d1)